gef-ui/server/controllers/application-details/index.test.js

```diff
@@ -352,27 +350,33 @@ describe('controllers/application-details', () => {
       it('renders `review-decision` when page requested is `review-decision`', async () => {
         mockApplicationResponse.status = CONSTANTS.DEAL_STATUS.UKEF_APPROVED_WITHOUT_CONDITIONS;
         api.getApplication.mockResolvedValueOnce(mockApplicationResponse);
-
-        await applicationDetails(MOCKS.MockRequestUrl('/gef/application/123/review-decision'), mockResponse);
+        const req = {
+          ...MOCKS.MockRequestUrl('/gef/application/123/review-decision'),
+          flash: jest.fn().mockReturnValue('Facility is updated'),
+        };
+        await applicationDetails(req, mockResponse);
 
         expect(mockResponse.render)
-          .toHaveBeenCalledWith('partials/review-decision.njk', expect.objectContaining({
-            applicationStatus: mockApplicationResponse.status,
-          }));
+          .toHaveBeenCalledWith('partials/review-decision.njk', expect.objectContaining({ applicationStatus: mockApplicationResponse.status }));
+        expect(req.flash).toHaveBeenCalledWith('success');
       });
 
       it('renders `unissued-facilities` when page requested is `unissued facilities` ', async () => {
         mockApplicationResponse.status = CONSTANTS.DEAL_STATUS.UKEF_ACKNOWLEDGED;
         api.getApplication.mockResolvedValueOnce(mockApplicationResponse);
-
-        await applicationDetails(MOCKS.MockRequestUrl('/gef/application/123/unissued-facilities'), mockResponse);
+        const req = {
```

Comment:
Can we add an assertion where `flash: jest.fn().mockReturnValue('')` or does not exists?

Reply:
Can this be a separate assertion please.

Reply:
Added separate assertion.